<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBiblePicture</class>
 <widget class="QDialog" name="DialogBiblePicture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>605</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>25</y>
     <width>156</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Click to Select:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>65</y>
     <width>151</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPictureInfo">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>5</y>
     <width>706</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labeTip">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>590</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right click on image to copy.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
